Dockerfile

```diff
@@ -1,16 +1,16 @@
 ###################
 # Build environment
 ###################
-FROM node:18.16-alpine AS builder
+FROM node:lts-bullseye-slim AS builder
```

Comment:
I think using versions is better so that we know for sure what image tag was used.